agent/tool-scripts/prometheus-metrics

```diff
@@ -145,7 +174,28 @@ case "$mode" in
 	start)
 	mkdir -p $tool_output_dir/json
 	while read -u 9 master; do
-		tool_cmd="$script_path/datalog/$tool-datalog $master $interval $tool_log"
+		# fetch keys from openshift master and set default values for cert and key vars if not defined
+		if [ -z "$cert" ]; then
+			if [ ! -s /run/pbench/keys/admin.crt ]; then
+				scp $master:/etc/origin/master/admin.crt /run/pbench/keys/admin.crt
+				if [[ $? != 0 ]]; then
+					error_log "[$script_name]scp command failed to copy the files from $master, please check if the cert exists in /etc/origin/master directory"
+					exit 1
+				fi
+			fi
+			cert=/run/pbench/keys/admin.crt
+		fi
+		if [ -z "$key" ]; then
+			if [ ! -s /run/pbench/keys/admin.key ]; then
+				scp $master:/etc/origin/master/admin.key /run/pbench/keys/admin.key
+				if [[ $? != 0 ]]; then
+					error_log "[$script_name]scp command failed to copy the files from $master, please check if the key exists in /etc/origin/master directory"
+					exit 1
+				fi	
+			fi
+			key=/run/pbench/keys/admin.key
+		fi
+		tool_cmd="$script_path/datalog/$tool-datalog $master $interval $tool_log $port $cert $key"
```

Comment:
@ndokos GOPATH stuff is tracked as a seperate PR - https://github.com/distributed-system-analysis/pbench/pull/610